docs/i18n/en/docusaurus-plugin-content-docs/current/05_ipn/01_config-ipn.md

```
@@ -6,6 +6,16 @@ When a payment is completed, VNPay will send an IPN (Instant Payment Notificatio
 
 **Note**: To set up the IPN URL for the production environment, you need to contact VNPay for support.
 
+### Quick Access (After Login)
+
+After logging into the VNPay merchant portal, you can directly access the Terminal configuration page (where IPN URL is set) by navigating to:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)